# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/dns/HTTPSSVC.h
# Commit: 96bd98319c62
# Full Hash: 96bd98319c6242a9b05dc4723b928e88537424e1
# Author: Kershaw Chang <kershaw@mozilla.com>
# Date: 2020-08-26 03:24:37
# Regressor Bug: 1652667
# File Overlap Count: 1
# Description:
#   Bug 1652667 - P1: Add telemetry for when an HTTPSSVC record is received r=dragana
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D87844
# ==============================================================================

diff -r b61a789151df -r 96bd98319c62 netwerk/dns/HTTPSSVC.h
--- a/netwerk/dns/HTTPSSVC.h	Tue Aug 25 17:29:21 2020 +0000
+++ b/netwerk/dns/HTTPSSVC.h	Tue Aug 25 17:29:32 2020 +0000
@@ -120,6 +120,8 @@
   already_AddRefed<nsISVCBRecord> GetServiceModeRecordInternal(
       bool aNoHttp2, bool aNoHttp3, const nsTArray<SVCB>& aRecords);
 
+  bool HasIPAddressesInternal(const nsTArray<SVCB>& aRecords);
+
   // The owner name of this HTTPS RR.
   nsCString mHost;
 };